File: public/examples/integrations/mcp-servers/mongodb/count_documents_example_call_tool.py
import json
from arcadepy import Arcade

client = Arcade()  # Automatically finds the `ARCADE_API_KEY` env variable

TOOL_NAME = "MongoDB.CountDocuments"

user_id = "{arcade_user_id}"

tool_input = {
    "database_name": "my_database",
    "collection_name": "users",
    "filter_dict": json.dumps({"status": "active"})
}

response = client.tools.execute(
    tool_name=TOOL_NAME,
    input=tool_input,
    user_id=user_id,
)

print(response.output.value)

File: public/examples/integrations/mcp-servers/mongodb/count_documents_example_call_tool.js
import { Arcade } from "arcadejs";

const client = new Arcade(); // Automatically finds the `ARCADE_API_KEY` env variable

const TOOL_NAME = "MongoDB.CountDocuments";

const userId = "{arcade_user_id}";

const toolInput = {
  database_name: "my_database",
  collection_name: "users",
  filter_dict: JSON.stringify({"status": "active"})
};

const response = await client.tools.execute({
  toolName: TOOL_NAME,
  input: toolInput,
  userId: userId,
});

console.log(response.output.value);
